contracts/contracts/vault/VaultStorage.sol

```diff
@@ -101,9 +101,9 @@ contract VaultStorage is Initializable, Governable {
     }
     /// @dev mapping of strategy contracts to their configuration
     // slither-disable-next-line uninitialized-state
-    mapping(address => Strategy) internal strategies;
+    mapping(address => Strategy) public strategies;
     /// @dev list of all vault strategies
-    address[] internal allStrategies;
+    address[] public allStrategies;
```

Comment:
We already have `getAllStrategies` on the ABI

Reply:
If we were starting fresh I'd agree with making `allStrategies` public and not having the `getAllStrategies` function.